frontend/src/components/Forms/Patient/PatientHelper.tsx

```diff
@@ -0,0 +1,176 @@
+	// 	ptMedicareCardExpiryDate: Yup.string().matches(
+	// 		/^(0[1-9]|1[0-2])-\d{4}$/,
+	// 		'Please enter a valid date in the format: MM-YYYY'
+	// 	),
+	// 	ptPrivateHealthFund: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptPrivateHealthFundNo: Yup.string().matches(
+	// 		/^[0-9]$/,
+	// 		'Only numbers are allowed for this field'
+	// 	),
+	// 	ptEmgContactGivenName: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptEmgContactSurname: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptEmgContactRelationship: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptEmgContactMobilePhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptEmgContactHomePhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptEmgContactWorkPhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptNextOfKinGivenName: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptNextOfKinSurname: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptNextOfKinRelationship: Yup.string().matches(
+	// 		/^[A-Za-z]+$/,
+	// 		'Only alphabets are allowed for this field'
+	// 	),
+	// 	ptNextOfKinMobilePhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptNextOfKinHomePhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptNextOfKinWorkPhone: Yup.string().matches(
+	// 		/^\+(?:[0-9] ?){6,14}[0-9]$/,
+	// 		'Please enter a valid phone number'
+	// 	),
+	// 	ptDVAFileNo: Yup.string().matches(
+	// 		/^\d{10}$/,
+	// 		'Only numbers are allowed for this field'
+	// 	),
+	// 	ptDVAExpiryDate: Yup.string().matches(
+	// 		// ^(0[1-9]|1[0-2])-\d{4}$
+	// 		/^(0[1-9]|1[0-2])-\d{4}$/,
+	// 		'Please enter a valid date in the format: MM-YYYY'
+	// 	),
+	// 	ptHealthcareCardNo: Yup.string().matches(
+	// 		/^\d{10}$/,
+	// 		'Only numbers are allowed for this field'
+	// 	),
+	// 	ptHealthcareCardExpiryDate: Yup.string().matches(
+	// 		/^(0[1-9]|1[0-2])-\d{4}$/,
+	// 		'Please enter a valid date in the format: MM-YYYY'
+	// 	),
+	// 	ptPensionCardNo: Yup.string().matches(
+	// 		/^\d{10}$/,
+	// 		'Only numbers are allowed for this field'
+	// 	),
+	// 	ptPensionCardExpiryDate: Yup.string().matches(
+	// 		// /^(0[1-9]|1[0-2])-\d{4}$/,
+	// 		// 'Please enter a valid date in the format: MM-YYYY'
+	// 		/^\d{10}$/,
+	// 		'Only numbers are allowed for this field'
+	// 	),
+	// });
```

Comment:
@ThanhNguyenn 

Did you temporarily comment out the validation schema for the patient's additional details or is this intentional? This disables the validation for the PatientAdditionalDetails component.